src/django/api/tiler.py

```diff
@@ -0,0 +1,62 @@
+from django.contrib.gis.geos import Polygon
+from django.db import connection
+from mercantile import bounds
+
+from api.querysets import get_facility_queryset_from_query_params
+
+
+def get_facilities_vector_tile(params, layer, z, x, y):
+    """
+    Create a vector tile for a layer generated via PostGIS's `ST_AsMVT`
+    function, filtered by params.
+
+    Arguments:
+    params (dict) -- Request query parameters whose potential choices are
+                     enumerated in `api.constants.FacilitiesQueryParams`
+    layer (string) -- The name of the tile layer. Currently only "facilities"
+                      is supported.
+    z (int) -- Zoom level.
+    x (int) -- X (horizontal) position for requested tile on a grid.
+    y (int) -- Y (vertical) position for requested tile on a grid.
+
+    Returns:
+    A vector tile.
+    """
+    tile_bounds = bounds(x, y, z)
+
+    mvt_geom_query = """
+        ST_AsMVTGeom(
+            location,
+            ST_MakeEnvelope({xmin}, {ymin}, {xmax}, {ymax}, 4326),
+            4096,
+            1000,
```

Comment:
We're going to drop this but the original rationale was a mistake in which I thought the units were meters rather than degrees.

Reply:
This 1000 buffer value being passed to `ST_AsMVTGeom` is in unit-less tile coordinates.

https://postgis.net/docs/manual-dev/ST_AsMVTGeom.html

Based on my reading of the PostGIS docs this is how far outside the tile bounds it will go and still encode features. This works in concert with the filter bbox. We learned that we need both to have both a tile buffer and a buffered bbox on the query to prevent clipping, but not a buffer on the actual bounds of the tile.

I saw tile clipping issues with the default of 256. They were resolved when I doubled it to 512.

Reply:
That makes sense -- I used 512 as the value in 
e160eba

Reply:
Sorry. I still saw some clipping at 512. 1024 may be more appropriate for our big (relative to the tile size) markers.

Reply:
Cool -- I just changed it to 1024